### PR TITLE
Solaris zfs changes

### PR DESCRIPTION
**Pull request by btoneill**
This gives more ZFS information that is useful

### CODE REVIEW

File: src/types.db [hunk 1]
```diff
@@ -14,7 +14,7 @@ cache_eviction		value:DERIVE:0:U
 cache_operation		value:DERIVE:0:U
 cache_ratio		value:GAUGE:0:100
 cache_result		value:DERIVE:0:U
-cache_size		value:GAUGE:0:U
+cache_size		value:GAUGE:0:1125899906842623
```
Comment (mfournier) [inline src/types.db]: Why change this ?

Comment (btoneill) [inline src/types.db]: The cache size value of U is a 32bit unsigned int. If you have a large
memory system, you overrun it and get no values. We ran into that with a
system that had 256G of memory.

Comment (btoneill) [inline src/types.db]: I picked that value as it was used in other places where you need a 64bit
counter....

**Comment by mfournier** [comment]
Many thanks @btoneill ! I just double-checked these values are also present on FreeBSD and ZFS-on-Linux, so hopefully we're safe regarding portability.

**Pull request merged**